lib/util/http.js

```diff
@@ -62,6 +62,15 @@ function http (url, opt) {
   if (url.indexOf('http') !== 0) {
     url = 'https:' + url
   }
+  if (process?.env.CI && process.env.FORWARDER_HOSTNAME && opt.headers) {
```

Comment:
Could we add some comments explaining what these lines do?
I realise it's pretty self explanatory, but to an automated tool or uneducated person reasoning the code it may be mis-identified as a backdoor re-directing requests (and authentication tokens)